code-reviews/pull-request-template.md

```diff
@@ -0,0 +1,28 @@
+### Link to the design document that prompted this PR?
+- Provide a link to the design document relevant to this area. 
+
+### What are you trying to address ?
+-  Describe the current behavior that you are modifying, and link to issue number
+
+### Description of new changes
+- Write a detailed description of all changes
+- If you have only done mechanical (not logical) changes point that out here
+-  Describe the new behavior that is introduces
+- If this introduces a breaking change, please describe the impact and migration path for existing applications below.
+
+### How did you test it?
+- Besides writing tests with your change, you should state the manual validation that you have done
+  * [ ] Ran it on specific OS
+  * [ ] Ran it using specific test set
+  * [ ] Description of test scenarios that you have tried.
+
+### Any relevant logs or outputs?
+
+- If you are printing something show a screenshot 
+- When you want to share long logs upload to:
+<br>(StorageAccount)/pr-support/attachments/(PR Number)/(yourFiles) using [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) or [portal.azure.com](portal.azure.com) and insert the link here.
```

Comment:
I would like to suggest a pass through this document with a markdown linter - like https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint for VS Code.

As an example this contains an open HTML br element that doesn't render well in all readers.  
It also starts with a ### 3rd level title, and a few more things like that.  

This comment applies to the readme.md as well.

Reply:
ran the markdown linter 

---
In reply to: [379418168](https://github.com/microsoft/code-with-engineering-playbook/pull/148#discussion_r379418168) [](ancestors = 379418168)